lib/enrollment_repository.rb

```diff
@@ -10,11 +10,20 @@ def initialize
   end
 
   def load_data(data_source)
+    data = {}
     filename = data_source.values.reduce({}, :merge!).values.join
     CSV.foreach(filename, headers: true, header_converters: :symbol) do |row|
       district_name = row[:location].upcase
-      @enrollment[district_name] ||= {}
-      @enrollment[dist_name][row[:timeframe].to_i] = clean_participation(row[:data])
+      data[district_name] ||= {}
+      data[dist_name][row[:timeframe].to_i] = clean_participation(row[:data])
+    end
+    create_enrollment_objects(data)
+  end
+
+  def create_enrollment_objects(enrollment_data)
```

Comment:
creates enrollment objects of district and data as hash within hash (needed to move forward with It_1